pkg/controller/propagator/propagation_test.go

```diff
@@ -26,14 +26,19 @@ func TestInitializeAttempts(t *testing.T) {
 		t.Run(
 			fmt.Sprintf(`%s="%s"`, attemptsEnvName, test.envVarValue),
 			func(t *testing.T) {
-				t.Parallel()
```

Comment:
are these changes related to this issue, i thought you are big on keepings seperate?

Reply:
@ckandag they are in a separate commit from the main chain. I needed it for the unit tests to pass in Prow.

Reply:
just want to note that :  these changes appear  to be  fixes/improvement to the test setup and are not really related specifically for the indent change.